docs/1.guide/components.md

```diff
@@ -118,6 +118,64 @@ The component is used to create a link to another page.
 ::
 <!-- prettier-ignore-end -->
 
+### Steps
+
+The component is used to create a list of steps that look pretty.
+
+<!-- prettier-ignore-start -->
+::tabs
```

Comment:
I pretty much prefer to find a way to “just” use simple md syntax. Like a numbered list with all having children .

 Docs are a tool for writing docs not frontend designer or undocs should add to the complexity only for the sake of design. The current syntax is not readable when rendered by github.

Reply:
That makes since! I appreciate the feedback.

Reply:
(BTW, we could still work on it, only probably to be coupled with a transform step to remove the need of markup.

Reply:
I wanted to close for now, I wanted to figure a good way to handle the transform then re-open PR.  I didn't know if we should trigger the change. I am thinking most numbered list might make since for the "Steps" component but in some cases they don't want too.

Reply:
You can read more about the tranform in https://github.com/unjs/undocs/blob/main/app/server/plugins/content.ts @cpreston321 

Example of GitHub flavoured blockquotes: https://github.com/unjs/undocs/blob/00d47631c3fb243ad15916327340742f14cd8f80/app/server/plugins/content.ts#L31-L44